### PR TITLE
Move messages component to global scope

### PR DESCRIPTION
**Pull request by donggyu04**
Move messages component to global scope, and access this component from `MessagesMixin`.

### CODE REVIEW

**Comment by songeunwoo** [comment]
LGTM!

File: ngrinder-frontend/src/js/store/vuex-store.js [hunk 1]
```diff
@@ -3,10 +3,14 @@ import Vuex from 'vuex';
...
         ngrinder: {},
+        messages: {},
     },
     mutations: {
         ngrinder(state, ngrinder) {
             state.ngrinder = ngrinder;
         },
+        messages(state, messages) {
+            state.messages = messages;
+        }
```
Comment (junoyoon) [inline ngrinder-frontend/src/js/store/vuex-store.js]: Shouldn't it be deleted?

File: ngrinder-frontend/src/js/main.js [hunk 2]
```diff
@@ -104,6 +105,7 @@ new Vue({
     components: {
         Copyright,
         Navigator,
+        Messages,
```
Comment (junoyoon) [inline ngrinder-frontend/src/js/main.js]: Deletable?

File: ngrinder-frontend/src/js/store/vuex-store.js (hunk 1 above)
Comment (donggyu04) [reply ngrinder-frontend/src/js/store/vuex-store.js]: Yes, it's unnecessary.

File: ngrinder-frontend/src/js/main.js (hunk 2 above)
Comment (donggyu04) [reply ngrinder-frontend/src/js/main.js]: It's necessary for this part
https://github.com/naver/ngrinder/pull/413/commits/1b5d0bd540018a2d64cd30ed52225583151bda98?file-filters%5B%5D=.js&file-filters%5B%5D=.vue#diff-d1f948ea1f38e5ed023c487a4447ab4cR16

**Pull request merged**